testapp/src/main/java/com/rakuten/tech/mobile/testapp/adapter/MiniAppListAdapter.kt

```diff
@@ -14,7 +14,7 @@ import com.rakuten.tech.mobile.miniapp.MiniAppInfo
 import com.rakuten.tech.mobile.miniapp.testapp.databinding.ItemListMiniappBinding
 
 
-class MiniAppListAdapter(var miniapps: List<MiniAppInfo>) :
+class MiniAppListAdapter(var miniapps: List<MiniAppInfo>, val iMiniAppList: IMiniAppList) :
```

Comment:
That looks like Hungarian Notation. I think in the current code base we don't have much of it. I think it is better to make all code look the same style.